### PR TITLE
Feature - Issue #505 - Develop branch to deploy to production AWS

### PR DESCRIPTION
**Pull request by richmahn**
For Issue #505, to deploy develop to the same AWS account as master, prefixing all develop items with `dev-`

### CODE REVIEW

**Review by phillip-hopper — DISMISSED** [review]
Can you point me to the Apex documentation for this? I found documentation for 
```
apex deploy --env-file ./some_file
```
but not 
```
apex deploy --env BRANCH_NAME
```

**Comment by richmahn** [comment]
@phillip-hopper: http://apex.run/#flag-env-file

**Comment by richmahn** [comment]
@phillip-hopper interestingly it doesn't have a section on the --env flag, but talks about it here: http://apex.run/#managing-infrastructure

File: .travis.yml [hunk 1]
```diff
@@ -3,9 +3,6 @@ python:
   - '2.7'
 
 before_install:
-  - eval export AWS_ACCESS_KEY_ID=\$${TRAVIS_BRANCH}_AWS_ACCESS_KEY_ID
```
Comment (richmahn) [inline .travis.yml:6]: These are now set without a branch prefix in Travis's settings, thus no need to set them here.

**Review by phillip-hopper — APPROVED** [review]
LGTM :+1:

**Pull request merged**